extend/flytekit-xgboost/flytekitplugins/xgboost/task.py

```diff
@@ -25,9 +26,9 @@ class ModelParameters(object):
         verbose_eval: If verbose_eval is True then the evaluation metric on the validation set is printed at each boosting stage.
     """
 
-    num_boost_round: int = 10
+    num_boost_round: int = 4
     early_stopping_rounds: Optional[int] = None
-    verbose_eval: Union[bool, int] = True
+    verbose_eval: Optional[Union[bool, int]] = True
```

Comment:
Why is this `Optional`? We have a default value set, correct?

Reply:
you are right